base/bitarray.jl

```diff
@@ -71,6 +71,28 @@ const BitMatrix = BitArray{2}
 
 BitVector() = BitArray{1}(undef, 0)
 
+"""
+    BitVector(nt::NTuple{N, Bool})
+
+Construct a `BitVector` from a tuple of `Bool`.
+# Examples
+```julia-repl
+julia> nt = (true, false, true, false)
+(true, false, true, false)
+
+julia> BitVector(nt)
+4-element BitArray{1}:
+ 1
+ 0
+ 1
+ 0
+```
+"""
+function BitVector(nt::NTuple{N, Bool}) where {N}
```

Comment:
This should be `Tuple{Vararg{Bool}}` to avoid forcing us to specialize it on every length.

Reply:
Done thanks